app/assets/stylesheets/application.scss

```diff
@@ -10,6 +11,4 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- *= require_tree .
- *= require_self
```

Comment:
Why do we need to remove these Sprockets directives?

Reply:
Because If I don't remove that, I would have problems using mixins